contracts/test/GrantRound.test.ts

```diff
@@ -149,46 +137,27 @@ describe('GrantRound', function () {
   });
 
   describe('Round end corner cases', () => {
-    before(async () => {
-      // End the round for all subsequent tests
+    it('sends remaining matching pool funds to payout address', async function () {
+      await roundContract.connect(mpUser).addMatchingFunds(ethers.utils.parseEther(balances[0]));
       await timeTravel(endTime + 1);
-    });
-
-    it.skip('sends remaining matching pool funds to payout address', async function () {
-      await mockERC20.mock.balanceOf.withArgs(grantPayee.address).returns(ethers.utils.parseEther(balances[0]));
-      await mockERC20.mock.balanceOf.withArgs(roundContract.address).returns(ethers.utils.parseEther(balances[0]));
-      await mockERC20.mock.transfer.withArgs(grantPayee.address, balances[0]).returns(true);
-
       await roundContract.connect(payoutAdmin).payoutGrants(grantPayee.address);
-
-      // TODO: add token.balanceOf check in mock ERC20
+      expect(await mockERC20.balanceOf(grantPayee.address)).to.be.equal(ethers.utils.parseEther(balances[0]));
     });
 
     it.skip('reverts if not the grant round payout administrator', async function () {
```

Comment:
Why are these tests skipped? I can't remember what the blocker is for getting these implemented

Reply:
Will be address in #50